lib/brasil_api/registrobr.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module BrasilAPI
+  class RegistroBR < Base
+    class << self
+      def whois(domain)
+        get("/registrobr/v1/#{domain}")
+      end
+    end
+  end
+end
```

Comment:
Adicionar a linha em branco no fim do arquivo:
```suggestion
end

```